xml/System.Data.OleDb/OleDbLiteral.xml

```diff
@@ -322,7 +322,7 @@
       </ReturnValue>
       <MemberValue>10</MemberValue>
       <Docs>
-        <summary>The character used in a LIKE clause to escape the character returned for the DBLITERAL_LIKE_UNDERSCORE literal. For example, if an underscore (\_) is used to match exactly one character and this is a backslash (\\), the characters "abc\\\_ _" match all character values that are five characters long and start with "abc\_". Some SQL dialects support a clause (the ESCAPE clause) that can be used to override this value. Maps to DBLITERAL_ESCAPE_UNDERSCORE_PREFIX.</summary>
+        <summary>The character used in a LIKE clause to escape the character returned for the DBLITERAL_LIKE_UNDERSCORE literal. For example, if an underscore (\_) is used to match exactly one character and this is a backslash (\\), the characters "abc\\\_" match all character values that are five characters long and start with "abc\_". Some SQL dialects support a clause (the ESCAPE clause) that can be used to override this value. Maps to DBLITERAL_ESCAPE_UNDERSCORE_PREFIX.</summary>
```

Comment:
The change should probably be like this: 

```suggestion
        <summary>The character used in a LIKE clause to escape the character returned for the DBLITERAL_LIKE_UNDERSCORE literal. For example, if an underscore (\_) is used to match exactly one character and this is a backslash (\\), the characters "abc\\\_ \_" match all character values that are five characters long and start with "abc\_". Some SQL dialects support a clause (the ESCAPE clause) that can be used to override this value. Maps to DBLITERAL_ESCAPE_UNDERSCORE_PREFIX.</summary>
```

Because this is what we wanna render:
![image](https://user-images.githubusercontent.com/12971179/59376733-8d827200-8d05-11e9-99a4-d3ac49c8aad4.png)

Reply:
Sure, applied your suggestion